direnvrc

```diff
@@ -111,6 +111,24 @@ _nix_argsum_suffix() {
   fi
 }
 
+_cache_is_valid() {
+  if [ "$(type -t nix_direnv_cache_is_valid)"  == "function" ]; then
+    nix_direnv_cache_is_valid || return 1;
```

Comment:
Looks good. Can you document `nix_direnv_cache_is_valid` in the README as well?

Reply:
Done in my last push.